kadena-signing-api/src/Kadena/SigningApi.hs

```diff
@@ -70,12 +70,12 @@ instance FromJSON SigningResponse where
   parseJSON = genericParseJSON compactEncoding
 
 newtype QuickSignRequest = QuickSignRequest
-  { _quickSignRequest_commands :: [CommandSigData]
+  { _quickSignRequest_reqs :: [CommandSigData]
```

Comment:
I see the reasoning behind this change.  Here are some alternative naming directions that I think could also make sense:

`_quicksignRequest...` (to match the URI change on line 100)

`..._sigDatas` or `..._commandSigDatas` (to emphasize the associated data type a little more)
`..._cmds` (to make more of a nod to Pact terminology)

I think these alternatives are worth a little consideration, although we should also time box this discussion as it doesn't impact functionality.

@alber70g @Randynamic Do you have any thoughts on this?

Reply:
IMHO I would go for `..._commandSigDatas`. Data type clarity is far more important, keeping the maintainability/ readability of the code in mind.

Reply:
Agreed.  I was also thinking about `cmds` but I like the more verbose thing as well.

Reply:
I'm all for full names. No abbreviations. For me even CommandSigData doesn't really tell what it is. The context here is QuickSignRequest, so I'd look for something that describes that.

Furthermore, I think there is some ambiguity between "command" (used for what's send to the `/send` endpoint consisting of hash, cmd, sigs) and the "cmd" (is inside the "command", that is the stringified payload).

[https://api.chainweb.com/openapi/pact.html#tag/endpoint-send/paths/~1send/post](openapi docs) talk about the "payload" as `Canonic non-malleable signed transaction data.`

I don't know how to make this non ambiguous. `transaction` seems the right word for `{ hash, signatures, cmd }`. But `cmd` doesn't cover the `serialized payload` because it contains a `cmd` which are the Pact expressions to be executed.

This was something I struggled with when defining my interfaces for the `@kadena/client`